cpp/include/ApprovalTests.v.7.0.0.hpp

```diff
@@ -0,0 +1,2928 @@
+// Approval Tests version v.7.0.0
```

Comment:
There's something I'm not understanding here... I'm finding that CMake downloads ApprovalTests, which I'd expect from the CMake... for example, in my debug build I get this:
`cpp/cmake-build-debug/approvaltests/ApprovalTests.v.7.0.0.hpp`
So I'm unsure why the ApprovalTests header also needs to be included in version control?

Reply:
Later: When I deleted this file, my build failed... So now I'm unsure of the purpose of the downloading via CMake...

Reply:
I had problems with getting meson to download and handle ApprovalTests. I didn't spend to much time on sorting the meson builds out, just added the files to version control instead of removing the meson build.

This file should not be used when building with CMake. The downloaded one in cmake-build... should be used. But there might be some CLion magic with the include folder name.

Reply:
> Later: When I deleted this file, my build failed... So now I'm unsure of the purpose of the downloading via CMake...

Apologies - I was wrong about this.

I think I must have accidentally deleted the file downloaded by cmake, rather than this version-controlled file...